### PR TITLE
[New Extension] Adding Support for Arc Appliance CLI Extension 

### PR DESCRIPTION
**Pull request by pkoushik**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by pkoushik** [comment]
Pushed out this PR for Visibility, please don't merge yet - will remove tag later.

**Comment by yungezz** [comment]
hi @pkoushik will sync you offline on process

**Comment by pkoushik** [comment]
@yungezz @fengzhou-msft would like to get some updates

File: src/index.json [hunk 1]
```diff
@@ -2948,6 +2948,60 @@
...
+                        "Programming Language :: Python :: 3.6",
+                        "Programming Language :: Python :: 3.7",
+                        "Programming Language :: Python :: 3.8",
+                        "License :: OSI Approved :: MIT License"
+                    ],
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "arcplatform-core@microsoft.com",
```
Comment (amanohar) [inline src/index.json:2973]: `appliance`

Comment (pkoushik) [reply src/index.json:3230]: updating in next release

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**